app/models/plant_accession.rb

```diff
@@ -5,4 +5,11 @@ class PlantAccession < ActiveRecord::Base
   has_many :plant_scoring_units
 
   include Annotable
+
+  validates :plant_accession,
+            presence: true
+
+  validates :year_produced,
+            presence: true,
+            length: {is: 4}
```

Comment:
This one is actually _not_ required by the migration. It is ok with me - but did you check all current DB values in this column fit that constraint?